includes/class-signature.php

```diff
@@ -191,6 +212,7 @@ protected static function check_legacy_key_pair_for( $user_id ) {
 	 * @return string The signature.
 	 */
 	public static function generate_signature( $user_id, $http_method, $url, $date, $digest = null ) {
+		_deprecated_function( __METHOD__, 'unreleased', self::class . '::sign_request()' );
```

Comment:
```suggestion
		\_deprecated_function( __METHOD__, 'unreleased', self::class . '::sign_request()' );
```

would slashing the `_deprecated_function` break the version replacement script?